### DISCUSSION TITLE
Failed to upload payment method icons.

### DISCUSSION THREAD

**Discussion by StarDust189** [ellite/Wallos#270] [Q&A]
I deployed Wallos in Docker on WSL using the Docker commands from the Readme. The icons I wanted to upload displayed correctly, but I encountered an error when saving。
![image](https://github.com/ellite/Wallos/assets/82697663/47757e15-dfcf-49c2-9314-9b673b3b1c3f)

I checked the address where the logo is stored, and the image was uploaded successfully. However, I cannot save it, and there are no error logs in Docker's logs. How can I resolve this?
![image](https://github.com/ellite/Wallos/assets/82697663/b8e03bc5-f2c1-4a00-9608-fc64dc3eea5b)

**Comment by ellite** [comment]
Can you open the network panel on your browser and tell me the response you see directly on the call? Thanks.

**Reply by StarDust189** [reply]
<br />
<b>Fatal error</b>:  Uncaught TypeError: imagecopyresampled(): Argument #2 ($src_image) must be of type GdImage, bool given in /var/www/html/endpoints/subscription/add.php:128
Stack trace:
#0 /var/www/html/endpoints/subscription/add.php(128): imagecopyresampled(Object(GdImage), false, 0, 0, 0, 0, 135, 26.289473684211, 190, 37)
#1 /var/www/html/endpoints/subscription/add.php(180): resizeAndUploadLogo(Array, '../../images/up...', 'BitzNet')
#2 {main}
  thrown in <b>/var/www/html/endpoints/subscription/add.php</b> on line <b>128</b><br />

**Reply by ellite** [reply]
Thank you. I'll have a look.
